src/test/groovy/com/calgaryscientific/gradle/VeracodeTaskTest.groovy

```diff
@@ -72,6 +72,11 @@ class VeracodeTaskTest extends Specification {
         System.out = stdout
     }
 
+    Node parseXMLString(String xml) {
```

Comment:
hm... it seems like we have lots of XMLIO in multiple places (veracodeTask, and now this) ... should we tract all of the XMLIO behavior to its own class? like XMLManipulation?

Reply:
Sounds good, but lets get that in another PR, this one is already getting big.